pkg/flagbuilder/build_flags.go

```diff
@@ -131,6 +131,12 @@ func BuildFlags(options interface{}) (string, error) {
 				flag = fmt.Sprintf("--%s=%s", flagName, vString)
 			}
 
+		case *string:
+			if v != nil {
+				vString := fmt.Sprintf("%v", *v)
```

Comment:
We are adding a case here, and we return error on unknown cases, so we know this is additive.

However, because we are no longer flattening `*string` to `string`, the `*string` block needs to be identical to the `string` block.

I think we need to do this through another tag (like `flag-empty`).  Perhaps a tag `flag-include-empty`?  The idea is that even if the flag is empty, we still want to send it.

If we can do that, and the `*string` and `string` blocks are then identical, this then LGTM.